templates/scripts/elasticsearch-logging.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+# This file wil run locally. It will run a run a job inside the kubernetes cluster that will set up indices and a lifecycle policy in Elasticsearch
+
+DOCKER_IMAGE_TAG=commitdev/zero-k8s-utilities:0.0.2
+ES_ENDPOINT=$(aws es describe-elasticsearch-domain --domain-name bill-aug21-stage-logging --query "DomainStatus.Endpoints.vpc" | jq -r '.')
```

Comment:
need to pass in variables from makefile 
https://github.com/commitdev/zero-deployable-backend/blob/master/Makefile#L21-L25

```suggestion
ES_ENDPOINT=$(aws es describe-elasticsearch-domain --domain-name $PROJECT-$ENVIRONMENT-logging --query "DomainStatus.Endpoints.vpc" | jq -r '.')
```